packages/web/src/components/gcds-fieldset/stories/gcds-fieldset.stories.tsx

```diff
@@ -37,73 +20,36 @@ export default {
         required: true,
       },
     },
-    disabled: {
+    legendSize: {
```

Comment:
We will need to set name so it displays as an attribute in the events and properties page.

```suggestion
    legendSize: {
      name: 'legend-size',
```